mappers/lodging.js

```diff
@@ -52,7 +52,7 @@ export default function mapLodgings(records, translations) {
       if (type) {
         publicG.add(sym(lodgingUri), RDF('type'), sym(type));
       } else {
-        console.error(`Cannot map information group value '${record['information_group']}' for record ${recordId}`);
+        errorLogger('information group', record['information_group'], recordId);
```

Comment:
Pass the same field name (`information_group`) as used to lookup the value as first argument. I know it was different before, but old code is wrong.